dev/com.ibm.ws.app.manager/resources/com/ibm/ws/app/manager/internal/resources/AppManagerMessages.nlsprops

```diff
@@ -216,3 +216,9 @@ APPLICATION_JANDEX_ENABLED.useraction=No action is required.
 APPLICATION_JANDEX_DISABLED=CWWKZ0065I: Jandex use is disabled for application {0}.
 APPLICATION_JANDEX_DISABLED.explanation=Jandex index is disabled for the specified application.
 APPLICATION_JANDEX_DISABLED.useraction=No action is required.
+
+#{0} is the application name
+error.startAfter.cycle=CWWKZ0066E: The following applications will not be started because the value of their startAfter attributes result in a cycle: {0} 
+error.startAfter.cycle.explanation=The dependencies described in the startAfter attributes of two or more applications result in a cycle that prevents those applications from starting.
```

Comment:
This explanation just restates information from the message. Can we give more insight? What about the startAfter values results in a cycle?

Reply:
I can't think of much that would be better here. I updated it to:
error.startAfter.cycle.explanation=None of the applications can start because each one depends on one of the other applications to be started first.

Reply:
that's good- thanks